### PR TITLE
ROX-24789: Make table borders consistent

### PR DESCRIPTION
**Pull request by sachaudh**
### Description

This PR removes the border in the expanded row for the CVEs table within the Request Details page. I also decided to update the table to use the TBodyUnified component.

### User-facing documentation

- [ ] CHANGELOG is updated
- [x] CHANGELOG update is not needed
- [ ] Documentation PR is created and linked above
- [x] Documentation is not needed

### Testing

- [x] inspected CI results

#### Automated testing

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests
- [x] contributed **no automated tests**

#### How I validated my change

Visual check of the UI

### CODE REVIEW

File: ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx [hunk 1]
```diff
@@ -112,34 +105,24 @@ function RequestCVEsTable({
...
-                    )}
-                    {data?.imageCVEs.length !== 0 &&
-                        data?.imageCVEs.map(
-                            (
-                                {
+                    <TbodyUnified
+                        tableState={tableState}
+                        colSpan={colSpan}
+                        emptyProps={{
+                            title: 'No CVEs found',
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx:112]: A comment in passing that this `title` has **found**
* which a reader might associate with filtering
* which differs from the pattern elsewhere in these changed files like **No pending exception requests**

File: ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx [hunk 2]
```diff
@@ -175,7 +158,7 @@ function RequestCVEsTable({
                                 );
 
                                 return (
-                                    <Tbody key={cve}>
+                                    <Tbody key={cve} isExpanded={isExpanded}>
```
Comment (pedrottimark) [inline ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx:161]: The `isExpanded={isExpanded}` prop for both `Tbody` and second `Tr` sounds like a nice learning (or annoyance, depending on your viewpoint) for team meeting next Friday.

**Review by pedrottimark — APPROVED** [review]
Clear changes, although I kept a close eye to see which removed the extra border.

File: ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx (hunk 1 above)
Comment (sachaudh) [reply ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx:112]: I see what you mean. Would it make more sense if it said `No CVEs in this request`? Technically, we should always have at least 1 CVE for a request so this should ideally not even show up.

Comment (pedrottimark) [reply ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/components/RequestCVEsTable.tsx]: Yes, that has a nice balance between not too short and not too long. Just right!

**Pull request merged**